#! /bin/sh /usr/share/dpatch/dpatch-run
## 04future.dpatch by  <az@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: 2.5 doesn't understand except with "as", only ","

@DPATCH@
diff -urNad duplicity-0.6.12~/duplicity duplicity-0.6.12/duplicity
--- duplicity-0.6.12~/duplicity	2011-03-12 20:27:59.000000000 +1000
+++ duplicity-0.6.12/duplicity	2011-03-12 20:35:23.807965909 +1000
@@ -877,7 +877,7 @@
         log.Notice(_("Deleting local %s (not authoritative at backend).") % del_name)
         try:
             os.unlink(del_name)
-        except Exception as e:
+        except Exception, e:
             log.Warn(_("Unable to delete %s: %s") % (del_name, str(e)))
 
     def copy_to_local(fn):
